src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -108,6 +108,16 @@ object DockerPlugin extends AutoPlugin {
     },
     dockerRmiCommand := dockerExecCommand.value ++ Seq("rmi"),
     dockerBuildCommand := dockerExecCommand.value ++ Seq("build") ++ dockerBuildOptions.value ++ Seq("."),
+    dockerAdditionalPermissions := {
+      val scripts = makeBashScripts.value
```

Comment:
I overlooked that. This causes a circular dependency between `JavaAppPackaging` and the `DockerPlugin`, which produces a bug reported in #1206